utils/src/test/kotlin/CopyrightStatementsProcessorTest.kt

```diff
@@ -43,4 +45,9 @@ class CopyrightStatementsProcessorTest : WordSpec() {
     }
 }
 
-private fun CopyrightStatementsProcessor.Result.toYaml(): String = yamlMapper.writeValueAsString(this)
+private fun CopyrightStatementsProcessor.Result.toYaml(): String =
+    yamlMapper.copy()
+        // Disable getter serialization without changing filed serialization.
```

Comment:
"filed" -> "field"?

Why don't we simply use `@JsonIgnore` in the `Result` class for `getAllStatements` like we do elsewhere?

Reply:
Fixed the type. I preferred this solution over `@JsonIgnore` as serialization is only needed for this test and won't be needed anywhere else it seems.

Reply:
Hmm, ok. It would have been nice to read about that reasoning in the commit message, but I'm fine with it.